source/mainnet/net/guides/company-identities.rst

```diff
@@ -9,202 +9,110 @@ Company identity creation
+            "v": 0,
+            "value": {
+               "accountKeys": {
+                  "keys": {
+                     "0": {
+                        "keys": {
+                           "0": {
+                              "signKey": "81e7d8e625a00b6f5b97dd8b0a97807212e6b0ceb4fd206e715b97536c83caea",
+                              "verifyKey": "03164c9e6654c1544a0e7d33780df425c695f6222fda75c047aea5186680e491"
+                           }
+                        },
+                        "threshold": 1
+                     }
+                  },
+                  "threshold": 1
+               },
+               "address": "3LfTBXYtc6TEjuJiKgLpFGEtGRMPhBsKRB76Q4x91LZPWSmQ9Z",
+               "credentials": {
+                  "0": "843785aef9446c8e5b2c6922863e49231b93fb9950909c3166e7c287357a1a495ecfbdcb6ca36ea5998fef2c9dee91f8"
+               }
+            }
+         }
 
-        user_cli recover-identity --cryptographic-parameters cryptographic-parameters-testnet.json --ip-info ip-info-testnet.json --request-out recovery-request.json
+      In this example the account has a single credential with index 0, and that credential has a single key with index 0. The private key is 03164c9e6654c1544a0e7d33780df425c695f6222fda75c047aea5186680e491 and its public key is 81e7d8e625a00b6f5b97dd8b0a97807212e6b0ceb4fd206e715b97536c83caea.
 
-      Email the ``recovery-request.json`` output file to support@concordium.software with the subject line "Recover company identity".
+   - ``address`` is the address of the account, e.g.,
 
-   #. When the recovery request has been validated successfully, Concordium will notify you by email, and they will return the identity object named ``id-object.json`` that you lost. With the recovered identity object, you can then recreate your account keys(account-keys.json) by running ``user_cli create-credential-v1``.
+      .. code-block:: json
 
-   If you experience issues, please contact Concordium’s technical support via support@concordium.software.
+         "address": "3LfTBXYtc6TEjuJiKgLpFGEtGRMPhBsKRB76Q4x91LZPWSmQ9Z"
 
 Import created accounts into ``concordium-client``
-==================================================
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+To use the accounts created in the Concordium Company ID tool, you must use Concordium Client. You can download it :ref:`here<concordium-node-and-client-download>`. For information about how to use it, see :ref:`concordium-client`.
 
 The account keys are primarily meant for clients to integrate into their key management solution and their software, e.g., an exchange integrating their trading platform with the Concordium chain.
 
-However if the ``account-keys.json`` file is not encrypted it can be imported into ``concordium-client`` with the command:
+However, if the ``account-keys.json`` file is not encrypted it can be imported into ``concordium-client`` with the command:
```

Comment:
In the command-line below, you should remove the `--format=genesis`. It is not longer needed (and also cannot be there).

Reply:
And in the line below that one remove the fact that `--name` is optional. It is mandatory now.